application/helpers/update/updatedb_helper.php

```diff
@@ -3721,6 +3721,41 @@ function ($v) {
             $oTransaction->commit();
         }
 
+        if ($iOldDBVersion < 441) {
+            $oTransaction = $oDB->beginTransaction();
+            // Convert old html editor modes if present in global settings
+            $oDB->createCommand()->update(
+                '{{settings_global}}',
+                array(
+                    'stg_value' => 'inline',
+                ),
+                "stg_name='defaulthtmleditormode' AND stg_value='wysiwyg'"
+            );
+            $oDB->createCommand()->update(
+                '{{settings_global}}',
+                array(
+                    'stg_value' => 'none',
+                ),
+                "stg_name='defaulthtmleditormode' AND stg_value='source'"
+            );
+            // Convert old html editor modes if present in profile settings
+            $oDB->createCommand()->update(
+                '{{users}}',
+                array(
+                    'htmleditormode' => 'inline',
+                ),
+                "htmleditormode='wysiwyg'"
+            );
+            $oDB->createCommand()->update(
+                '{{users}}',
+                array(
+                    'htmleditormode' => 'none',
+                ),
+                "htmleditormode='source'"
+            );
+            $oDB->createCommand()->update('{{settings_global}}', array('stg_value' => 438), "stg_name='DBVersion'");
```

Comment:
438? :O

Reply:
Typo after moving the code for a new db version

Reply:
I've done this 100 times. When we refactor each update into its own class, dbversion will be part of base class and never touched again, making typos like this impossible.